play/src/main/resources/evalution/default/2.sql

```diff
@@ -0,0 +1,79 @@
+# --- !Ups
+
+CREATE TABLE IF NOT EXISTS Users (
```

Comment:
Nope. Normally this conflicts will be solved by play-slick evolution. But since we turned it off, you probably need to delete it manually